packages/core/contracts/Escrow.sol

```diff
@@ -182,8 +182,20 @@ contract Escrow is IEscrow, ReentrancyGuard {
     }
 
     /**
-     * @dev Bulk payout workers
-     * Should fail if any of the transaction is failing.
+     * @notice Bulk payout workers
+     * Escrow needs to be complted / cancelled, so that it can be paid out.
+     * Every recipient is paid with the amount after reputation and recording oracle fees taken out.
+     * If the amount is less than the fee, the recipient is not paid.
+     * If the fee is zero, reputation, and recording oracle are not paid.
+     * Payout will fail if any of the transaction fails.
+     * If the escrow is fully paid out, meaning that the balance of the escrow is 0, it'll set as Paid.
+     * If the escrow is partially paid out, meaning that the escrow still has remaining balance, it'll set as Partial.
+     *
+     * @param _recipients Array of recipients
+     * @param _amounts Array of amounts
+     * @param _url URL of the results
+     * @param _hash Hash of the results
+     * @param _txId Transaction ID
      */
```

Comment:
Optional(add) : This contract is only callable it the contract is not broke, not launched, not paid, not expired, by trusted parties.